apps/web/src/pages/User/styles.ts

```diff
@@ -21,11 +21,10 @@ export const StyledUserContainer = styled('div', {
   alignItems: 'center',
   flexDirection: 'column',
   padding: 26,
+  width: '100%',
 });
 
 export const StyledProjectContainer = styled('div', {
-  display: 'flex',
-  flexDirection: 'row',
-  gap: 20,
-  padding: 26,
+  width: '100%',
+  padding: '40px 100px',
```

Comment:
is this mobile friendly?

Reply:
oh no.. I don't know what happened there